Misc/NEWS.d/next/Library/2018-10-17-16-32-00.bpo-30274.g0iI1.rst

```diff
@@ -0,0 +1,2 @@
+Rename 'name' to 'fullname' argument to
```

Comment:
Typo: I think "argument" should come after `name`.

Also, we use \* to format arguments.

```rst
Rename *name* argument to *fullname* ...
Patch by Sayan Chowdhury.
```